robolectric/src/main/java/org/robolectric/util/ActivityController.java

```diff
@@ -178,6 +188,22 @@ public void run() {
       }
     });
 
+    ViewRootImpl root = component.getWindow().getDecorView().getViewRootImpl();
+    ShadowViewRootImpl shadow = (ShadowViewRootImpl) ShadowExtractor.extract(root);
+    Display display = Shadow.newInstanceOf(Display.class);
+    ShadowDisplay shadowDisplay = Shadows.shadowOf(display);
+    Rect frame = new Rect(0, 0, shadowDisplay.getWidth(), shadowDisplay.getHeight());
+    Rect insets = new Rect(0, 0, 0, 0);
```

Comment:
Do you know where these values come from in real android? I think its probably OK to start with dummy values for now though.